HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -63,6 +63,33 @@ def customiseFor20429(process):
         del producer.GBRForestFileName
     return process
 
+
+# 
+# The three different set of thresholds will be used to study
+# possible new thresholds of pfrechits and effects on high level objects
+# The values proposed (A, B, C) are driven by expected noise levels
+#
```

Comment:
What does this comment have to do with the customisation function added in the following? There is nothing about A B C ....